pom.xml

```diff
@@ -417,6 +417,38 @@
                 <module>build-time-analytics</module>
             </modules>
         </profile>
+        <profile>
+            <id>exclude-podman</id>
```

Comment:
We will have to add this property every time we run on Docker? What about jobs which divide modules between different profiles?